src/components/authenticated/AuthenticatedNavBar.tsx

```diff
@@ -9,21 +9,32 @@ import "../NavBar.scss";
 
 export function AuthenticatedNavBar() {
 	return (
-		<Navbar expand="lg" fixed="bottom" className="Nav">
-			<Container className="d-flex justify-content-around Nav_container">
-				<SignOutButton />
-				<Nav.Link as={NavLink} to="/" className="Nav__link" aria-label="Home">
-					Home
-				</Nav.Link>
-
-				<Nav.Link
-					as={NavLink}
-					to="/manage-list"
-					className="Nav__link"
-					aria-label="Manage List"
-				>
-					Manage List
-				</Nav.Link>
+		<Navbar
+			collapseOnSelect
+			expand="md"
+			className="bg-secondary rounded-bottom-3 pt-0"
+		>
+			<Container>
+				<Navbar.Brand className="bg-primary rounded-bottom-3 text-center mb-3 px-3 text-dark fw-bolder shadow">
+					GrocerEase
+				</Navbar.Brand>
+				<Navbar.Toggle aria-controls="basic-navbar-nav" />
+				<Navbar.Collapse>
+					<Nav className="flex-grow-1 justify-content-evenly align-items-center fw-bolder p-1 text-center">
+						<Nav.Link as={NavLink} to="/" aria-label="Home" eventKey={"1"}>
+							Home
+						</Nav.Link>
+						<Nav.Link
+							as={NavLink}
+							to="/manage-list"
+							aria-label="Manage List"
+							eventKey={"2"}
+						>
+							Manage List
+						</Nav.Link>
+					</Nav>
+					<SignOutButton />
```

Comment:
I was thinking out loud!!! What if we had the sign out button outside the hamburger menu ? The user would then need 1 click to sign out, instead of 2 clicks (clicking hamburger menu, then sign out?). Now...I'm not sure how involved that would be. Just a thought!